src/middleware/errorHandler.js

```diff
@@ -15,8 +15,10 @@ const logRequestInfosInErrorCase = (error, req, res, next) => {
 		}
 
 		requestError(req, (decodedJWT || {}).userId, error);
+		next(error);
+	} else {
+		next();
 	}
-	next(error);
 };
 
 const formatAndLogErrors = (showRequestId) => (error, req, res, next) => {
```

Comment:
if error makes no sense in this case the functions should not be called without error instead

Reply:
change to if(!error){return next();}

Reply:
i fix it